nixos/modules/module-list.nix

```diff
@@ -1320,6 +1320,7 @@
   ./services/web-apps/dolibarr.nix
   ./services/web-apps/engelsystem.nix
   ./services/web-apps/ethercalc.nix
+  ./services/web-apps/firefly-iii.nix
```

Comment:
Module addition needs a release entry, as per the PR checklist

Reply:
Added